### PR TITLE
feat!: Make the ReviewService usable from netstandard2.0 projects.

### PR DESCRIPTION
**Pull request by jeanplevesque**
## Proposed Changes

 - [ ] Bug fix
 - [x] Feature
 - [ ] Code style update (formatting)
 - [x] Refactoring (no functional changes, ~~no api changes~~)
 - [ ] Build or CI related changes
 - [x] Documentation content changes
 - [ ] Other, please describe:

## What is the current behavior?

We can't reference `ReviewService` from netstd2.0 projects, even though it's completely platform agnostic.

## What is the new behavior?

We can reference `ReviewService` from netstd2.0 projects.

### Packages refactoring
There are now 3 packages:
- ReviewService.Abstractions (contains interfaces and building blocks)
  - `netstandard2.0` (unchanged)
- ReviewService (contains netstd2.0-compatible implementations, including `ReviewService`)
  - `netstandard2.0` (was previously `net6.0;net6.0-android;net6.0-ios;net6.0-windows10.0.19041`)
- ReviewService.NativePrompters (contains the platform specific `IReviewPrompter` implementations)
  - `net6.0;net6.0-android;net6.0-ios;net6.0-windows10.0.19041`

### Other changes
- Added `LoggingReviewPrompter` as a netstd2.0-compatible implementation of `IReviewPrompter`.
- Renamed a few files with extension methods.
- Linked the README for each csproj.

## Checklist

Please check that your PR fulfills the following requirements:

- [x] Documentation has been added/updated.
- [ ] Automated tests for the changes have been added/updated.
- [ ] Updated [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) (if you introduced a breaking change).
      I did not update BREAKING_CHANGES.md because the package is not considered stable yet.

### CODE REVIEW

File: src/ReviewService.Abstractions/ReviewService.Abstractions.csproj [hunk 1]
```diff
@@ -14,13 +14,20 @@
 		<Copyright>https://github.com/nventive/ReviewService/blob/main/LICENSE</Copyright>
 		<RootNamespace>ReviewService</RootNamespace>
 	</PropertyGroup>
+	<ItemGroup>
```
Comment (Soap-141) [inline src/ReviewService.Abstractions/ReviewService.Abstractions.csproj:19]: We should add a space here.

File: src/ReviewService.NativePrompters/ReviewService.NativePrompters.csproj [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		<PackageId>ReviewService.NativePrompters</PackageId>
+		<Description>Native implementations of IReviewPrompter interface from the ReviewService.Abstractions package.</Description>
+		<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+		<PackageProjectUrl>https://github.com/nventive/ReviewService</PackageProjectUrl>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>review;maui;winui;ios;android</PackageTags>
+		<Copyright>https://github.com/nventive/ReviewService/blob/main/LICENSE</Copyright>
+		<RootNamespace>ReviewService</RootNamespace>
+	</PropertyGroup>
+	<ItemGroup>
```
Comment (Soap-141) [inline src/ReviewService.NativePrompters/ReviewService.NativePrompters.csproj:19]: Space here to keep it clean.

File: src/ReviewService/LoggingReviewPrompter.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace ReviewService;
+
+/// <summary>
+/// Implementation of <see cref="IReviewPrompter"/> that only logs the <see cref="TryPrompt"/> invocation.
+/// </summary>
+public class LoggingReviewPrompter : IReviewPrompter
```
Comment (Soap-141) [inline src/ReviewService/LoggingReviewPrompter.cs]: Let's seal this class.

File: src/ReviewService/ReviewService.csproj [hunk 4]
```diff
@@ -14,6 +14,13 @@
 		<Copyright>https://github.com/nventive/ReviewService/blob/main/LICENSE</Copyright>
 		<RootNamespace>ReviewService</RootNamespace>
 	</PropertyGroup>
+	<ItemGroup>
```
Comment (Soap-141) [inline src/ReviewService/ReviewService.csproj:19]: Add a space here too.

**Review by Soap-141 — CHANGES_REQUESTED** [review]

**Review by Soap-141 — APPROVED** [review]

**Review by takla21 — APPROVED** [review]

**Pull request merged**